data_dictionary.md

```diff
@@ -0,0 +1,37 @@
+
+# Example Activity Stream Log
+
+{"session_duration":1635,"locale":"en-US","ip":"10.192.171.13","date":"2016-03-07",
+  "unload_reason":"close","client_id":"374dc4d8-0cb2-4ac5-a3cf-c5a9bc3c602e","max_scroll_depth":145,"addon_version":"1.0.0",
+  "total_history_size":9,"ver":"3","ua":"python-requests\/2.9.1","click_position":"100x100","source":"top_sites",
+  "timestamp":1457396660000,"action":"activity_stream","tab_id":3,"load_reason":"restore","total_bookmarks":19}
+  
+The `about:newtab` page will ping (HTTPS POST) [Onyx][https://github.com/mozilla/onyx] every time the page loses focus.  
+
+* `session_duration` is defined to be the time in milliseconds between the newtab gaining and losing focus :one:
+* `locale` is the browser chrome's language (eg. en-US) :two:
+* `ip` is the IP address of the client :two:
+* `date` is the date in YYYY-MM-DD format :three:
+* `unload_reason` is the reason the about:newtab page lost focus :one:
+* `client_id` is an identifier for this client :one:
+* `max_scroll_depth` is the maximum number of pixels the scroll bar was dragged in this session :one:
+* `addon_version` is the version of the Activity Stream addon :one:
+* `total_history_size` is the number of history items currently in the user's places db :one:
+* `ver` is the version of the Onyx API the ping was sent to :one:
+* `ua` is the user agent string :two:
+* `click_position` is the index of the element that was clicked :one:
+* `source` is either (recent_links, recent_bookmarks, frecent_links, top_sites, spotlight) and indicates what was clicked :two:
+* `timestamp` is the time in ms since epoch :three:
+* `action` is always `activity_stream` :one:
+* `tab_id` is the FX generated unique id for the tab :one:
+* `load_reason` is either (newtab, refocus, restore) and is the reason the tab was focused :one:
+* `total_bookmarks` is the total number of bookmarks in the user's places db :one:
```

Comment:
nit: Not sure if these bullets/keys are in a specific order, or if we feel like A-Zing for readability, or grouping by :one: => :two: => :three: so they are grouped more logically.